pycaret/timeseries.py

```diff
@@ -0,0 +1,2223 @@
+from typing import Optional
+
+
+def setup(
+    data,
+    target,
+    train_size: float = 0.7,
+    numeric_features: Optional[list] = None,
+    numeric_imputation="mean",
+    date_features: Optional[list] = None,
+    experiment_name: Optional[str] = None,  # mlflow tracking
+    ignore_features: Optional[list] = None,
+    log_data: bool = False,  # mlflow tracking
+    log_experiment: bool = False,  # mlflow tracking
+    log_plots: bool = False,  # mlflow tracking
+    log_profile: bool = False,  # mlflow tracking
+    normalize: bool = False,
+    normalize_method: str = "zscore",
+    transformation: bool = False,
+    transformation_method: str = "yeo-johnson",
+    remove_outliers: bool = False,  # new
+    outliers_threshold: float = 0.05,  # new
+    session_id: Optional[int] = None,
+    transform_target: bool = False,  # new
+    transform_target_method: str = "box-cox",  # new
+    verbose: bool = True,
+    silent: bool = False,
+    profile: bool = False,
+):
+
+    """
+        
+    Description:
+    ------------    
+    This function initializes the environment in pycaret and creates the transformation
+    pipeline to prepare the data for modeling and deployment. setup() must called before
+    executing any other function in pycaret. It takes two mandatory parameters:
+    dataframe {array-like, sparse matrix} and name of the target column. 
+    
+
+    Parameters
+    ----------
+    * data : {array-like, sparse matrix}, shape (n_samples, n_features) where n_samples 
```

Comment:
Please fix the docstrings to follow numpy standard.
https://numpydoc.readthedocs.io/en/latest/format.html

Reply:
@Yard1 fixed. It actually likes prettier following Numpy doc guidelines